### PR TITLE
Fixed some issues with the zone labels

### PR DESCRIPTION
**Pull request by Mopplikus**
This PR fixes some issues with the zone labels, namely the fact that previously the label wouldn't work properly when pressed directly, and only showed when the zone polygon was clicked. Also, the labels now show the current owner of the zone as well.

### CODE REVIEW

**Comment by Jenjamin3000** [comment]
When I tried the app, the owner is only showed when we attack a zone. Is this normal?

**Comment by Jenjamin3000** [comment]
> When I tried the app, the owner is only showed when we attack a zone. Is this normal?

This is alright! I do not why I did not see before...

File: app/src/androidTest/java/com/github/campus_capture/bootcamp/MapVoteTest.java [hunk 1]
```diff
@@ -180,10 +180,10 @@ public void timerbuttonIsDisplayedWhenOutsideOfTakeover() throws InterruptedExce
 
         Intents.init();
 
-        onView(ViewMatchers.withContentDescription("Navigate up"))
-                .perform(ViewActions.click());
+        //onView(ViewMatchers.withContentDescription("Navigate up"))
```
Comment (Jenjamin3000) [inline app/src/androidTest/java/com/github/campus_capture/bootcamp/MapVoteTest.java:183]: Is this not making the test useless?

File: app/src/main/java/com/github/campus_capture/bootcamp/fragments/MapsFragment.java [hunk 2]
```diff
@@ -159,50 +159,68 @@ public class MapsFragment extends Fragment implements GoogleMap.OnCameraMoveList
...
 
         // scheduler.startAll();
     };
 
+    /**
+     * Private method to refresh the current attacks / owner on the label of any of the maps' markers
+     * @param m the marker to be affected
+     * @param map the google maps instance to move the camera
+     */
+    private void refreshCurrentAttacks(Marker m, GoogleMap map)
```
Comment (Jenjamin3000) [inline app/src/main/java/com/github/campus_capture/bootcamp/fragments/MapsFragment.java:188]: Nice refactor! And the owners are know displayed! Good job!

**Review by Jenjamin3000 — APPROVED** [review]
This works just fine! Alright for me. Nice modularization!

File: app/src/androidTest/java/com/github/campus_capture/bootcamp/MapVoteTest.java (hunk 1 above)
Comment (Mopplikus) [reply app/src/androidTest/java/com/github/campus_capture/bootcamp/MapVoteTest.java:183]: No no, the initial screen is already the app, and previously we would load it again to trigger the mock back-end. But here it wasn't necessary somehow (thank the CI for this change)

File: app/src/main/java/com/github/campus_capture/bootcamp/fragments/MapsFragment.java (hunk 2 above)
Comment (Mopplikus) [reply app/src/main/java/com/github/campus_capture/bootcamp/fragments/MapsFragment.java:188]: Thanks :)

**Pull request merged**